antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/ANTLRFileStream.kt

```diff
@@ -15,19 +13,19 @@ import org.antlr.v4.kotlinruntime.misc.Utils
  */
 @Deprecated("as of 4.7 Please use {@link CharStreams} interface.")
 class ANTLRFileStream
-private constructor(override val sourceName: String, val charset: Charset = UTF8)
+private constructor(override val sourceName: String)
```

Comment:
Shouldn't we consider the Charset?

Reply:
yes, but adding an big external dependency (that is not yet kotlin 1.4 ready) just for the charset is another thing...  because we can load a file in the client code and pass it to antlr-kotlin as a string we can skip the encoding problem.